### ISSUE TITLE
LEED Summary Report special end-use subcategory tags are not documented and some problems with the EAp2-4/5 report

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5364]
Helpdesk Ticket 10982
User asked how to get a value in the LEED Summary Report EAp2-4/5 subreport row labeled "Fans-Parking Garage".  I didn't see anything mentioned in the Input Output Reference or in the  Using EnergyPlus for Compliance doc.  Found [this section](https://github.com/NREL/EnergyPlus/blob/9da112c653f018f8484435fada1ff1cf68aa6593/src/EnergyPlus/OutputReportTabular.cc#L7476-L7508) in the source code and created an example file by adding these objects that use the special end-use subcategory tags.

```
ElectricEquipment,
    Kitchen Electric,        !- Name
    SPACE1-1,                !- Zone or ZoneList Name
    EQUIP-1,                 !- Schedule Name
    EquipmentLevel,          !- Design Level Calculation Method
    2000,                    !- Design Level {W}
    ,                        !- Watts per Zone Floor Area {W/m2}
    ,                        !- Watts per Person {W/person}
    0.3,                     !- Fraction Latent
    0.3,                     !- Fraction Radiant
    0.2,                     !- Fraction Lost
    Cooking;                 !- End-Use Subcategory

ElectricEquipment,
    Vertical Transportation, !- Name
    SPACE2-1,                !- Zone or ZoneList Name
    OCCUPY-1,                !- Schedule Name
    EquipmentLevel,          !- Design Level Calculation Method
    1000,                    !- Design Level {W}
    ,                        !- Watts per Zone Floor Area {W/m2}
    ,                        !- Watts per Person {W/person}
    0,                       !- Fraction Latent
    0,                       !- Fraction Radiant
    0,                       !- Fraction Lost
    Elevators and Escalators;!- End-Use Subcategory

ElectricEquipment,
    Industrial Equip,        !- Name
    SPACE3-1,                !- Zone or ZoneList Name
    EQUIP-1,                 !- Schedule Name
    EquipmentLevel,          !- Design Level Calculation Method
    3000,                    !- Design Level {W}
    ,                        !- Watts per Zone Floor Area {W/m2}
    ,                        !- Watts per Person {W/person}
    0,                       !- Fraction Latent
    0.2,                     !- Fraction Radiant
    0.2,                     !- Fraction Lost
    Industrial Process;      !- End-Use Subcategory

Lights,
    Process Lighting,        !- Name
    SPACE3-1,                !- Zone or ZoneList Name
    EQUIP-1,                 !- Schedule Name
    LightingLevel,           !- Design Level Calculation Method
    500,                     !- Lighting Level {W}
    ,                        !- Watts per Zone Floor Area {W/m2}
    ,                        !- Watts per Person {W/person}
    0,                       !- Return Air Fraction
    0.2,                     !- Fraction Radiant
    0.2,                     !- Fraction Visible
    1,                       !- Fraction Replaceable
    Interior Lighting - Process,  !- End-Use Subcategory
    No;                      !- Return Air Fraction Calculated from Plenum Temperature

GasEquipment,
    Gas Cooking Equip,       !- Name
    SPACE1-1,                !- Zone or ZoneList Name
    EQUIP-1,                 !- Schedule Name
    EquipmentLevel,          !- Design Level Calculation Method
    5000,                    !- Design Level {W}
    ,                        !- Power per Zone Floor Area {W/m2}
    ,                        !- Power per Person {W/Person}
    0.2,                     !- Fraction Latent
    0.3,                     !- Fraction Radiant
    0.3,                     !- Fraction Lost
    ,                        !- Carbon Dioxide Generation Rate {m3/s-W}
    Cooking;                 !- End-Use Subcategory

GasEquipment,
    Gas Process Equip,       !- Name
    SPACE3-1,                !- Zone or ZoneList Name
    EQUIP-1,                 !- Schedule Name
    EquipmentLevel,          !- Design Level Calculation Method
    1000,                    !- Design Level {W}
    ,                        !- Power per Zone Floor Area {W/m2}
    ,                        !- Power per Person {W/Person}
    0.2,                     !- Fraction Latent
    0.3,                     !- Fraction Radiant
    0.3,                     !- Fraction Lost
    ,                        !- Carbon Dioxide Generation Rate {m3/s-W}
    Industrial Process;       !- End-Use Subcategory

Exterior:Lights,
    Exterior Lighting,       !- Name
    Always1,                 !- Schedule Name
    5000,                    !- Design Level {W}
    AstronomicalClock,       !- Control Option
    This Does Not Matter for the LEEED Report;  !- End-Use Subcategory

Schedule:Constant,Always1,,1;

Exterior:FuelEquipment,
    Parking Garage Fans,     !- Name
    Electricity,             !- Fuel Use Type
    OCCUPY-1,                !- Schedule Name
    500,                     !- Design Level {W}
    Fans - Parking Garage;   !- End-Use Subcategory

```

Most items landed in the correct spot, but noticed that the exterior lighting row moved to the top and was labeled in all caps, and that the cooking and process electric and gas loads are included in the Receptacle Equipment as well as the Cooking and Industrial Process rows.

Screen shot of the table before adding new objects with the special end-use subcategories.
![image](https://cloud.githubusercontent.com/assets/4991075/11702113/92c39df6-9e99-11e5-9a17-be9d714d401b.png)

And after adding those objects
![image](https://cloud.githubusercontent.com/assets/4991075/11702123/ac72c5e2-9e99-11e5-9a27-88d2b00889bf.png)

**Comment by mjwitte** [comment]
Closed via #5580

**Comment by xwanguk** [comment]
Sorry but I have a question regarding the report after these End-Use Subcategory settings and simulation. For example, if I input "Fans - parking garage" for ElectricEquipment for multiple zones, can I access each zone output such as "Fans - parking garage:InteriorEquipment:Electricy" (I mean from the eso file) in addition to the figure that comes from the total building (summation of all zones that have this input)?
